.github/workflows/oppiabot.yml

```diff
@@ -0,0 +1,21 @@
+# Contains jobs corresponding to Oppiabot.
+
+name: Oppiabot
+
+on:
+  pull_request_target:
+    branches:
+      - develop
```

Comment:
On what branches should the CLA check be enabled?

Reply:
As we need CLA (Oppiabot) for all the Pull request to take action, how should we mention the branch?
Also, will it work for fork branches as well?

Reply:
When you mention "develop" here: https://github.com/oppia/oppia-android/blob/f8a5a936c8d9adf8f1ada6227362d223b3cb5604/.github/workflows/oppiabot.yml#L5-L8

It means that the CLA Check will run on any PR that has the develop branch as its base.
Do we need the check to run on all branches?
And could you rephrase "Also, will it work for fork branches as well?"?? I'm not able to understand?

Reply:
This yml file is using secrets, which PRs from fork might not have access, so will the CLA works as we required?

Reply:
Yes, the PR's from a fork will be able to access the secrets since it runs on the event. `pull_request_target`. ([Source](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target))